app/assets/javascripts/backbone/views/calendar_view.js.coffee

```diff
@@ -0,0 +1,51 @@
+class MS.Views.CalendarView extends Backbone.View
+
+  el: '.calendar'
+
+  initialize: (params) ->
+    # Initialize calendar
+
+    @$calendar = @$('#calendar')
+
+    @$calendar.fullCalendar({
+      # Changes the default event render to load in html rather than title only
+      eventRender: (calEvent, element) ->
+        element.find('.fc-title').html(calEvent.title)
+
+      customButtons: {
+        legend: {
+          text: 'Legend'
+        }
+      },
```

Comment:
I don't think you need these curly braces. CoffeeScript should infer them based on the indent.